src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -66,6 +66,37 @@ public function getDataFormSpecifications()
         return $tests;
     }
 
+    public function testLineBreaksBeforeDocumentSeparatorAndComment()
+    {
+        $yaml = <<<'EOF'
+
+ 
+# Two empty lines above (second - with one whitespace) and two below (first - with three whitespaces).
+   
+
```

Comment:
You can just put everything on one line am use the `\n` escape sequence for line breaks.